grpc/channelz/v1/channelz.proto

```diff
@@ -435,9 +436,14 @@ service Channelz {
 }
 
 message GetTopChannelsRequest {
-  // start_channel_id indicates that only channels at or above this id should be
+  // start_channel_id indicates that only channels above this id should be
   // included in the results.
   int64 start_channel_id = 1;
+
+  // If non-zero, the server will return a page of results whose
+  // cardinality is equal to or less than this value. If zero, the
```

Comment:
nit: cardinality is a big word that some none-english speakers may not understand.  Consider picking simpler, less apt words.  (Chrome even highlights it as misspelled for me).

Reply:
Changed to be easier to understand.